Mathlib/LinearAlgebra/Quotient/Basic.lean

```diff
@@ -220,11 +218,7 @@ theorem mapQ_pow {f : M →ₗ[R] M} (h : p ≤ p.comap f) (k : ℕ)
   induction' k with k ih
   · simp [LinearMap.one_eq_id]
   · simp only [LinearMap.iterate_succ]
-    -- Porting note: why does any of these `optParams` need to be applied? Why didn't `simp` handle
-    -- all of this for us?
-    convert mapQ_comp p p p f (f ^ k) h (p.le_comap_pow_of_le_comap h k)
-      (h.trans (comap_mono <| p.le_comap_pow_of_le_comap h k))
-    exact (ih _).symm
+    rw [mapQ_comp, ih]
```

Comment:
Does the porting note still apply? Is it wontfix/not worth tracking?

Reply:
I guess it is slightly mysterious why we can't combine the `rw` into the `simp`. But I think this is minor, and not needing the huge convert is satisfactory.